<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;宋体&quot;;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>707</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_N"/>
    <addaction name="action_O"/>
    <addaction name="action_S"/>
    <addaction name="action_A"/>
    <addaction name="separator"/>
    <addaction name="action_U"/>
    <addaction name="action_P"/>
    <addaction name="separator"/>
    <addaction name="action_X"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_U_2"/>
    <addaction name="separator"/>
    <addaction name="action_T"/>
    <addaction name="action_C"/>
    <addaction name="action_P_2"/>
    <addaction name="action_L"/>
    <addaction name="separator"/>
    <addaction name="action_F"/>
    <addaction name="action_N_2"/>
    <addaction name="action_R"/>
    <addaction name="action_G"/>
    <addaction name="separator"/>
    <addaction name="action_A_2"/>
    <addaction name="action_D"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>格式(&amp;O)</string>
    </property>
    <addaction name="action_W"/>
    <addaction name="action_F_2"/>
    <addaction name="action_C_2"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>查看(&amp;V)</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_S_2"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_H"/>
    <addaction name="separator"/>
    <addaction name="aboutSorftworeAction"/>
    <addaction name="aboutQtAction"/>
    <addaction name="websiteAction"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_O"/>
   <addaction name="menu_V"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="action_O"/>
   <addaction name="action_S"/>
   <addaction name="action_U_2"/>
   <addaction name="action_T"/>
   <addaction name="action_C"/>
   <addaction name="action_P_2"/>
   <addaction name="separator"/>
   <addaction name="action_L"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_N">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/图标.png</normaloff>:/new/icon/images/图标.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="action_O">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/open.png</normaloff>:/new/icon/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)...</string>
   </property>
  </action>
  <action name="action_S">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/save.png</normaloff>:/new/icon/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="action_A">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/saveas.png</normaloff>:/new/icon/images/saveas.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)...</string>
   </property>
  </action>
  <action name="action_U">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/set.png</normaloff>:/new/icon/images/set.png</iconset>
   </property>
   <property name="text">
    <string>页面设置(&amp;U)..</string>
   </property>
  </action>
  <action name="action_P">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/print.png</normaloff>:/new/icon/images/print.png</iconset>
   </property>
   <property name="text">
    <string>打印(&amp;P)...</string>
   </property>
  </action>
  <action name="action_X">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/exit.png</normaloff>:/new/icon/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;X)...</string>
   </property>
  </action>
  <action name="action_U_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/cancel.png</normaloff>:/new/icon/images/cancel.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
  </action>
  <action name="action_T">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/da962553d1f438b4450b2d93da65516a.jpg</normaloff>:/new/icon/images/da962553d1f438b4450b2d93da65516a.jpg</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
  </action>
  <action name="action_C">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/copy.png</normaloff>:/new/icon/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="action_P_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/paste.png</normaloff>:/new/icon/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
  </action>
  <action name="action_L">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/delete.png</normaloff>:/new/icon/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除(&amp;L)</string>
   </property>
  </action>
  <action name="action_F">
   <property name="text">
    <string>查找(&amp;F)...</string>
   </property>
  </action>
  <action name="action_N_2">
   <property name="text">
    <string>查找下一个(&amp;N)</string>
   </property>
  </action>
  <action name="action_R">
   <property name="text">
    <string>替换(&amp;R)...</string>
   </property>
  </action>
  <action name="action_G">
   <property name="text">
    <string>转到(&amp;G)...</string>
   </property>
  </action>
  <action name="action_A_2">
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
  </action>
  <action name="action_D">
   <property name="text">
    <string>时间/日期(&amp;D)</string>
   </property>
  </action>
  <action name="action_W">
   <property name="text">
    <string>√自动换行(&amp;W)</string>
   </property>
  </action>
  <action name="action_F_2">
   <property name="text">
    <string>字体(&amp;F)...
</string>
   </property>
  </action>
  <action name="action_S_2">
   <property name="text">
    <string>状态栏(&amp;S)</string>
   </property>
  </action>
  <action name="action_H">
   <property name="text">
    <string>查看帮助(&amp;H)</string>
   </property>
  </action>
  <action name="aboutSorftworeAction">
   <property name="text">
    <string>关于记事本(&amp;A)</string>
   </property>
  </action>
  <action name="action_C_2">
   <property name="text">
    <string>颜色(&amp;C)
</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="text">
    <string>关于Qt(&amp;Q)</string>
   </property>
  </action>
  <action name="websiteAction">
   <property name="text">
    <string>官网(&amp;W)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
